plugins/azure-devops-backend/src/api/PullRequestsDashboardProvider.ts

```diff
@@ -115,6 +110,8 @@ export class PullRequestsDashboardProvider {
     const dashboardPullRequests =
       await this.azureDevOpsApi.getDashboardPullRequests(projectName, options);
 
+    await this.getAllTeams(); // Make sure team members are loaded
```

Comment:
Perhaps not ideal the first time but it's called in the other methods so I guess it's fine :)